viper.py

```diff
@@ -512,10 +516,25 @@ def fit_chunk(o, chunk, obsname, targ=None, tpltarg=None):
 
             if tplname:
                 p, e_p = S_mod.fit(x_ok, f_ok, v, a, bg, s,t, c=cc, c0=c0, dx=0.1, sig=sig[i_ok])
+            elif createtpl:
+                p, e_p = S_mod.fit(x_ok, f_ok, v, a, bg, s,t, c=cc, c0=c0, dx=0.1, sig=sig[i_ok], res=False, notell= True)
             else:
             # do not fit for velocity
                 p, e_p = S_mod.fit(x_ok, f_ok, a=a, b=bg, s=s,t=t0, c=cc, v0=0, c0=c0, dx=0.1, sig=sig[i_ok])
 
+    if createtpl:
+        S_tell = S_mod(x, *p)
+        S_tell /= np.nanmean(S_tell)
+
+        tf = f/S_tell
+        tf[S_tell<0.2] = 1
```

Comment:
Add comments on variables `S_tell` and `tf` (is it a "telluric corrected spectrum"?).
`f/S_tell` => `f / S_tell`
`tf[S_tell<0.2] = 1` does it mean that too strong contaminated lines are set to unity?

Reply:
Ok.
Again `nan` is less confusing than `1`.